packages/remark-iframes/src/index.js

```diff
@@ -31,6 +31,18 @@ module.exports = function plugin (opts) {
     }
     return finalUrl
   }
+  function computeThumbnail (provider, url) {
+    let thumbnail = 'default image'
+    if (provider.thumbnail && provider.thumbnail.format) {
+      thumbnail = provider.thumbnail.format
+      Object.keys(provider.thumbnail).filter(key => key !== 'format')
+        .forEach(function (key) {
+          thumbnail = thumbnail.replace(/`{${key}}`/g,
+            new RegExp(provider.thumbnail[key]).exec(url)[1])
+        })
+    }
+    return thumbnail
+  }
```

Comment:
I think it's really strange

`const replace = new RegExp(thumbnailConfig[key]).exec(url)` means "in url search the value that we found on the config with the given key". It will never find anything.